File: fact.js
let number = [];
let userInp = parseInt(prompt("Enter the Number to find Factorial!!"));
for (let i = 1; i <= userInp; i++) {
  number[i - 1] = i;
}
const fact = number.reduce((prv, curr) => {
  return prv * curr;
});
console.log(`Factorial of ${userInp} is ${fact}`);

File: fact.cpp
#include <iostream>

using namespace std;

int main()
{
    int fact = 1;
    int num;
    cout<<"Enter a number to find factorial: ";
    cin>>num;
    for(int i = 1 ; i <= num ; i++){
        fact = fact * i;
    }
    cout<<"Factorial of the " << num << " is: "<<fact;

    return 0;
}
